src/components/fieldset/_fieldset.scss

```diff
@@ -4,12 +4,22 @@
 
 @include govuk-exports("govuk/component/fieldset") {
   .govuk-fieldset {
+    min-width: 0;
     margin: 0;
     padding: 0;
     border: 0;
     @include govuk-clearfix;
   }
 
+  // Fix for Firefox < 53
```

Comment:
Given Firefox 53 was released in April 2017 and we're now on version 66 I _think_ we can probably omit this? On GOV.UK, 6.18% of Firefox users were using a version older than 53, which represents about 0.29% of the total traffic.

Thoughts?

Reply:
@36degrees Honestly I'll leave this one down to you and the team. I was stung many times in my career by Firefox fieldset oddities so maybe I'm emotionally attached 😁

Reply:
Actually, whilst the default browser was changed to IE11 a few months ago, I believe DWP Jobcentre work coaches and other staff still have access to Firefox 45 ESR.

It's included in various automated tests.

Is that reason to keep it? Thanks